CONTRIBUTING.md

```diff
@@ -84,10 +84,10 @@ We use the following [labels](https://github.com/plotly/plotly.js/labels) to tra
   [`package-lock.json`](https://docs.npmjs.com/files/package-lock.json) file is
   used and updated correctly.
 
-#### Step 1: Clone the plotly.js repo and install its dependencies
+#### Step 1: Clone the plotly.js repo, download it and install dependencies
```

Comment:
I think this should be "fork, clone" instead of "clone, download"

Reply:
Resolved in https://github.com/plotly/plotly.js/pull/5791/commits/113c99642bc5fb239f5e7e0db8e6638d678966cc.